packages/inference/src/tasks/custom/streamingRequest.ts

```diff
@@ -66,6 +66,9 @@ export async function* streamingRequest<T>(
 			if (done) return;
 			onChunk(value);
 			for (const event of events) {
+				if (options?.signal?.aborted) {
```

Comment:
It's probably more correct to pass `signal` into `fetch` to explicitly abort connection (adding it to `info` in `makeRequestOptions`).
And then handle AbortError here (or let it bubble to user, not sure).

Reply:
yes totally right, I realize something was wrong when I did the VCR 🥲 I was expecting like 2 tokens, and kept getting the 1000 tokens, while the stream was interrupt, the fetch kept going :)